Kuroba/app/src/main/java/com/github/adamantcheese/chan/core/site/sites/chan420/Chan420.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Kuroba - *chan browser https://github.com/Adamantcheese/Kuroba/
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.github.adamantcheese.chan.core.site.sites.chan420;
+
+import androidx.annotation.NonNull;
+
+import com.github.adamantcheese.chan.core.model.Post;
+import com.github.adamantcheese.chan.core.model.orm.Board;
+import com.github.adamantcheese.chan.core.model.orm.Loadable;
+import com.github.adamantcheese.chan.core.site.ChunkDownloaderSiteProperties;
+import com.github.adamantcheese.chan.core.site.Site;
+import com.github.adamantcheese.chan.core.site.SiteIcon;
+import com.github.adamantcheese.chan.core.site.common.CommonSite;
+import com.github.adamantcheese.chan.core.site.common.taimaba.TaimabaActions;
+import com.github.adamantcheese.chan.core.site.common.taimaba.TaimabaApi;
+import com.github.adamantcheese.chan.core.site.common.taimaba.TaimabaCommentParser;
+import com.github.adamantcheese.chan.core.site.common.taimaba.TaimabaEndpoints;
+import com.github.adamantcheese.chan.utils.Logger;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import okhttp3.HttpUrl;
+
+public class Chan420 extends CommonSite {
+    private final ChunkDownloaderSiteProperties chunkDownloaderSiteProperties;
+    private static final String TAG = "420Chan";
+
+    public static final CommonSiteUrlHandler URL_HANDLER = new CommonSiteUrlHandler() {
+        private final String[] mediaHosts = new String[]{"boards.20chan.org"};
```

Comment:
Is it really supposed to be "20chan" here?

Reply:
good catch!